runtime/bindings/python/tests/io_test.py

```diff
@@ -54,7 +54,9 @@ def compile_mm_test():
     global MM_TEST_COMPILED
     if not MM_TEST_COMPILED:
         MM_TEST_COMPILED = iree.compiler.compile_str(
-            MM_TEST_ASM, target_backends=iree.compiler.core.DEFAULT_TESTING_BACKENDS
+            MM_TEST_ASM,
+            target_backends=iree.compiler.core.DEFAULT_TESTING_BACKENDS,
+            extra_args=["--iree-opt-const-eval=false"],
```

Comment:
```suggestion
            # TODO(#15984): re-enable const eval once parameters are supported.
            extra_args=["--iree-opt-const-eval=false"],
```

Reply:
(or whatever issue is tracking it - I just picked your issue, but looks like that's what breaks it)